raiden/tests/fixtures/blockchain.py

```diff
@@ -320,6 +330,12 @@ def blockchain_services(
 
 
 @pytest.fixture
+def registry(blockchain_services, registry_address):
+    blockchain_services.deploy_service
```

Comment:
This statement does nothing. I guess it's a typo and you want to combine both lines so that the return works.